backend/apps/nest/constants.py

```diff
@@ -0,0 +1,13 @@
+"""Nest app constants."""
+
+ISSUES_INDEX = 5
+GITHUB_COM_INDEX = 2
+MIN_PARTS_LENGTH = 4
+
+DEADLINE_FORMAT_ERROR = "Invalid deadline format. Use YYYY-MM-DD or YYYY-MM-DD HH:MM."
+DEADLINE_FUTURE_ERROR = "Deadline must be in the future."
+FETCH_ISSUE_ERROR = "Failed to fetch issue from GitHub: {error}"
+INVALID_ISSUE_LINK_FORMAT = "Invalid GitHub issue link format."
+ISSUE_LINK_ERROR = "Issue link must belong to an OWASP repository."
+PRICE_POSITIVE_ERROR = "Price must be a positive value."
+PRICE_VALID_ERROR = "Price must be a valid number."
```

Comment:
Narrow down the scope -- add to a separate constants only something that you need in multiple modules.